### PR TITLE
rm gh token

### PR DESCRIPTION
**Pull request by d-roak**
This change is [Reviewable](https://reviewable.io/reviews/kkrt-labs/kakarot/988)

### CODE REVIEW

File: scripts/artifacts.py [hunk 1]
```diff
@@ -68,8 +68,9 @@ def get_artifacts(
     name: Union[str, Path] = Path("resources"), base_branch_name: str = "main"
 ):
     # Pull latest main artifacts
+    # https://api.github.com/repos/kkrt-labs/kakarot-ssj/releases/latest | jq -r '.assets[0].browser_download_url'
     response = requests.get(
-        f"https://api.github.com/repos/kkrt-labs/kakarot/actions/artifacts?name={name}&per_page=50"
+        f"https://api.github.com/repos/krt-labs/kakarot/actions/artifacts?name={name}&per_page=50"
```
Comment (Eikix) [inline scripts/artifacts.py:73]:
```suggestion
        f"https://api.github.com/repos/kkrt-labs/kakarot/actions/artifacts?name={name}&per_page=50"
```

**Review by Eikix — COMMENTED** [review]
lgtm, @d-roak just needs to fix artifacts.py and ef-test.py, and then we can review again

@ClementWalter wdyt

File: docker/deployer/Dockerfile [hunk 2]
```diff
@@ -15,9 +15,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     jq \
     && rm -rf /var/lib/apt/lists/*
 
-ARG GITHUB_TOKEN
```
Comment (d-roak) [inline docker/deployer/Dockerfile:18]: Are the scripts called within the docker file?

**Comment by d-roak** [comment]
dependency on this + new latest release
https://github.com/kkrt-labs/kakarot-ssj/pull/741

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**